modules/serverless-creating-subscriptions-yaml.adoc

```diff
@@ -6,12 +6,12 @@
 [id="serverless-creating-subscriptions-yaml_{context}"]
 = Creating a subscription by using YAML
 
-You can create a subscription to connect a channel to a sink by using YAML.
+After you have created a channel and an event sink, you can create a subscription to enable event delivery. Creating Knative resources by using YAML files uses a declarative API, which enables you to describe subscriptions declaratively and in a reproducible manner. To create a subscription by using YAML, you must create a YAML file that defines a `Subscription` object, then apply it by using the `oc apply` command.
```

Comment:
Same here with _files uses_ and second sentence.

Reply:
See other similar comment response